src/Form/formatters.js

```diff
@@ -0,0 +1,102 @@
+const {FOUR_DIGIT_YEAR_REGEX, DATE_SEPARATOR_REGEX} = require('./selectors')
```

Comment:
Moved these to a separate files because we now have a few of them. The new stuff starts at line 46.